fluid/understand_sentiment_by_dynrnn.py

```diff
@@ -0,0 +1,193 @@
+import argparse
+import cPickle
+import os
+import random
+
+import numpy
+import paddle.v2.dataset.imdb as imdb
+import paddle.v2.fluid as fluid
+from paddle.v2 import batch
+
+try:
+    with open('word_dict.pkl', 'r') as f:
+        word_dict = cPickle.load(f)
+except:
+    word_dict = imdb.word_dict()
+    with open('word_dict.pkl', 'w') as f:
+        cPickle.dump(word_dict, f, cPickle.HIGHEST_PROTOCOL)
+
+
+def cache_reader(reader, clean):
```

Comment:
We do not have a unified way to accelerate our benchmark demos. I doubt about the reasonability of caching the dataset with CPickle when do benchmark.